### PR TITLE
Milestone1: A clone of Youtube Video Player

### PR DESCRIPTION
**Pull request by digitekh2i**
In this embedding images and video project, the features include:

1. Header navigation section
2. Main video player section
3. Aside section

### CODE REVIEW

**Comment by elmejdki** [comment]
## Pull Request is under review by Zakariae El Mejdki

File: index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                   <div><p>57,800 Views * june 9, 2020</p></div>
+                   <div><i class="fas fa-thumbs-up"></i></div>
+                   <div><i class="fas fa-thumbs-down"></i></div>
+                   <div class="baby-flex"><i class="fas fa-share"></i><div>share</div></div>
+               </div>
+        </div>
+        <div class="listedView">
+            <div class="iconcue"><img src="image/firstfoto.jpg" alt=""></div> 
+            <div><img src="image/img5.jpg" alt="" class="iconcue"></div>
+            <div><img src="image/img4.jpg" alt="" class="iconcue"></div>
```
Comment (elmejdki) [inline index.html:47]: Man you can do that, you should add your own CSS and HTML not getting captures from the original page

**Review by elmejdki — CHANGES_REQUESTED** [review]
# Hello @digitekh2i nice job so far :clap:, but I think you can make it much better.:tophat:

- Please fix the links of the images on the navbar:
image

- Also, I see from your readme image that they are not well aligned, please align them exactly as they are on the original page:
image

- Please align the like and dislike buttons with the views section you can do it easily with flexbox:
### your version:
:point_down: :point_down: :point_down: 
image
### Original
:point_down: :point_down: :point_down: 
image

- add also the number of likes and dislikes, it should look the same as the original.

### Please make all the requested the changes here :point_up: and also here :point_down:, before requesting another code review.
_if you need any kind of help don't hesitate to contact me on slack._

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required ♻️ 
****
### Hi @digitekh2i  and @name! 👋 
You have done a great job implementing the changes requested by the previous TSE! 👍 You have also done a wonderful job with your project so far! It looks really good! 👏 👏 👏  There are just a few issues that need to be resolved before you can move on: ⬇️ 
****

- [ ] The HTML validator is showing some errors. Please use [this validator](https://validator.w3.org) to check your HTML file for errors and fix them.
Screen Shot 2020-06-16 at 6 39 13 PM

- [ ] The elements of your navigation bar are not properly aligned. There is too much space between the YouTube logo and the hamburger icon as well as the search input box and search icon. Please compare your clone with the original page in the screenshots below and fix this. 
The original page 👇 
Screen Shot 2020-06-16 at 6 49 34 PM
Your clone 👇 
Screen Shot 2020-06-16 at 6 49 43 PM

- [ ] The section with the like, share, save buttons and the views and date are not properly aligned. There is not enough space between the likes/date and the buttons. Please compare your clone with the original page and fix this.
The original page 👇 
Screen Shot 2020-06-16 at 6 54 19 PM 
Your clone 👇 
Screen Shot 2020-06-16 at 6 54 25 PM






****
### Please make the required changes and resubmit for a new code review once you are ready 🙏

Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

**Comment by santiago-rodrig** [comment]
# PR under review by @santiago-rodrig

File: index.html [hunk 2]
```diff
@@ -8,43 +8,114 @@
...
-            <div><img src="image/img4.jpg" alt="" class="iconcue"></div>
+            <div class="privcons">
+                <div id="upNext" ><a href="#">Up Next</a></div>
+                <div id="autoP" ><a href="#">AUTOPLAY</a></div>
+                <div id="toggle-btn" ><a href="#"><i class="fas fa-toggle-on"></i></a></div>
+            </div>
+            <div class="listed-video-style">
+                <div><iframe src="https://www.youtube.com/embed/YJZCUhxNCv8" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></div>
+                <div><span class="aside-titles" id="info-style-big4"><a href="">  How to get a job at the big 4 - Amazon, Faceboo, Google &...</a></span>
+                    <br>
```
Comment (santiago-rodrig) [inline index.html:74]: Don't use `<br />` like this, it is semantically incorrect, it is only intended to be used inside paragraphs or quotes.

File: index.html [hunk 3]
```diff
@@ -8,43 +8,114 @@
...
-            <div class="iconcue"><img src="image/firstfoto.jpg" alt=""></div> 
-            <div><img src="image/img5.jpg" alt="" class="iconcue"></div>
-            <div><img src="image/img4.jpg" alt="" class="iconcue"></div>
+            <div class="privcons">
+                <div id="upNext" ><a href="#">Up Next</a></div>
+                <div id="autoP" ><a href="#">AUTOPLAY</a></div>
+                <div id="toggle-btn" ><a href="#"><i class="fas fa-toggle-on"></i></a></div>
+            </div>
+            <div class="listed-video-style">
+                <div><iframe src="https://www.youtube.com/embed/YJZCUhxNCv8" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></div>
```
Comment (santiago-rodrig) [inline index.html:72]: Video previews shouldn't be displayed, putting some placeholder images will do for this project.

File: style.css [hunk 4]
```diff
@@ -1,90 +1,381 @@
...
 body{
 background-color: #fffffe;
-width: 100;
-margin: auto;
+margin: 0px;
+border: 0px;
 font-family: Arial, Helvetica, sans-serif;
 }
 
 text{
```
Comment (santiago-rodrig) [inline style.css:10]: this is not even an HTML element.

File: style.css [hunk 5]
```diff
@@ -1,90 +1,381 @@
...
-}
-nav{
-    width: 100%;
+#header-box{
+display: flex;
+justify-content: space-evenly;
+padding: 10px 5px;
+height:55px;
+align-items: center;
+border-bottom: 20px;
```
Comment (santiago-rodrig) [inline style.css:20]: Increasing the padding could work too.

**Review by santiago-dev96 — CHANGES_REQUESTED** [review]
# Changes required 🔧

You are getting close to completion! Amazing job so far. 👍

Yet, there are some things to fix.

1. The search button is being rendered as an input of type text, I suppose that you wanted to use an input of type submit instead

2. The video previews (right sidebar) shouldn't be playable, just using images for those is enough

Please submit another CRR after implementing the changes.

You can find me in slack as **srodrig**, happy coding! 👋

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — COMMENTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @digitekh2i, a great project, and great job :+1: You fixed all but one comment made by the previous TSE:slightly_smiling_face: I would re-highlight what the previous TSE's commented so it can be addressed and approved. Kindly address comment below:

- We currently have tons of ```<br>``` tags in our code. We were advised to not use them, margin or padding could be used instead 

Kindly fix this minor issue and submit another code review :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
### Status: Change required :recycle:  

### Hello @digitekh2i    :hand: 

### Good job, you did great job with the project :+1: 

### There is one small thing that must be addressed before we can go further. You have to maintain a professional github repository in all your project, that includes a good title and description to your pull request:
Screenshot from 2020-06-18 10-37-40
I think this title is not good, please provide instead a good title (ex: Youtube clone, or video player clone ....) and also a good description where you talk about what this pull request add (ex: This project is a clone of the video player of youtube we added ....).
### Once done with the changes, please ask for another code review

**Review by elmejdki — APPROVED** [review]
# Great job @digitekh2i :clap: your project looks exactly the same as the original page :tophat:
### status: Approved

you can merge now to master your project is approved.

### Happy Coding

**Comment by digitekh2i** [comment]
> # Great job @digitekh2i 👏 your project looks exactly the same as the original page 🎩
> ### status: Approved
> you can merge now to master your project is approved.
> 
> ### Happy Coding

Thank You!

**Comment by digitekh2i** [comment]
> ### Status: Change required ♻️
> ### Hello @digitekh2i ✋
> ### Good job, you did great job with the project 👍
> ### There is one small thing that must be addressed before we can go further. You have to maintain a professional github repository in all your project, that includes a good title and description to your pull request:
> Screenshot from 2020-06-18 10-37-40
> I think this title is not good, please provide instead a good title (ex: Youtube clone, or video player clone ....) and also a good description where you talk about what this pull request add (ex: This project is a clone of the video player of youtube we added ....).
> 
> ### Once done with the changes, please ask for another code review

Thank you!

**Comment by digitekh2i** [comment]
> ## Status: Require changes ♻️ 🚥
> Hi @digitekh2i, a great project, and great job 👍 You fixed all but one comment made by the previous TSE🙂 I would re-highlight what the previous TSE's commented so it can be addressed and approved. Kindly address comment below:
> 
> * We currently have tons of `<br>` tags in our code. We were advised to not use them, margin or padding could be used instead
> 
> Kindly fix this minor issue and submit another code review 😄
> _Happy coding_ 👨‍💻
> [Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

Thank you!

**Comment by digitekh2i** [comment]
> # Changes required 🔧
> You are getting close to completion! Amazing job so far. 👍
> 
> Yet, there are some things to fix.
> 
> 1. The search button is being rendered as an input of type text, I suppose that you wanted to use an input of type submit instead
> 2. The video previews (right sidebar) shouldn't be playable, just using images for those is enough
> 
> Please submit another CRR after implementing the changes.
> 
> You can find me in slack as **srodrig**, happy coding! 👋

Thank you!

**Comment by digitekh2i** [comment]
> ### Status: Changes Required ♻️
> ### Hi @digitekh2i and @name! 👋
> You have done a great job implementing the changes requested by the previous TSE! 👍 You have also done a wonderful job with your project so far! It looks really good! 👏 👏 👏 There are just a few issues that need to be resolved before you can move on: ⬇️
> 
> * [ ]  The HTML validator is showing some errors. Please use [this validator](https://validator.w3.org) to check your HTML file for errors and fix them.
>   Screen Shot 2020-06-16 at 6 39 13 PM
> * [ ]  The elements of your navigation bar are not properly aligned. There is too much space between the YouTube logo and the hamburger icon as well as the search input box and search icon. Please compare your clone with the original page in the screenshots below and fix this.
>   The original page 👇
>   Screen Shot 2020-06-16 at 6 49 34 PM
>   Your clone 👇
>   Screen Shot 2020-06-16 at 6 49 43 PM
> * [ ]  The section with the like, share, save buttons and the views and date are not properly aligned. There is not enough space between the likes/date and the buttons. Please compare your clone with the original page and fix this.
>   The original page 👇
>   Screen Shot 2020-06-16 at 6 54 19 PM
>   Your clone 👇
>   Screen Shot 2020-06-16 at 6 54 25 PM
> 
> ### Please make the required changes and resubmit for a new code review once you are ready 🙏
> Good Luck and Happy Coding 🍀 👩🏾‍💻
> 
> _P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

Thank you

**Pull request merged**